lib/package-lock.json

```diff
@@ -0,0 +1,1015 @@
+{
```

Comment:
I would prefer to run `yarn` instead and keep one single lock file, as there's a `yarn.lock` already. In the future, once esy supports PnP, we might be able to do everything with esy.

Reply:
Yeah, `esy.lock` needs to be commited, but I think the current setup you have with yarn.lock inside `lib` is perfect 👌  until we revisit the packaging / dependencies handling with `esy` in the future.